docs/CHANGEMAKER_DATA.md

```diff
@@ -13,6 +13,13 @@ endpoints. Changemaker data cannot currently be directly written to the PDC,
 though that functionality is on the roadmap. Instead, changemaker data is
 aggregated from any proposals that have been submitted by the changemaker.
 
+## Registering fiscal sponsors
+
+To register one changemaker as a fiscal sponsor of another changemaker
+explicitly (in addition to the proposal data reflecting fiscal sponsors), use
+`PUT /changemakers/:changemakerId/fiscalSponsors/:fiscalSponsorChangemakerId`
```

Comment:
I see that `fiscalSponsors` is right there in the API route.  But we're not treating fiscal sponsorship specially, as a type of relationship, right?  For example, if we were to define "sharing staff" as a type of relationship, then would there be an endpoint `/changemakers/:changemakerId/sharesStaff/:sharedStaffChangemakerId`?  I.e., for every arbitrary relationship we define, part of making that definition is choosing an endpoint name?

Reply:
The task in #1485, as I understood it, was to explicitly model fiscal sponsorship in our entity model. If there is another task in the future to "explicitly model sharing staff in our entity model" then yes we would need to add endpoints to allow for that relationship to be modeled.

Did we miss the mark?

Reply:
I would not say we missed the mark based on the current use case and scope of work @bickelj -- if that happens in future we can implement support for generic entity relationships, but for now we have one very specific kind of relationship to model.

So to @kfogel's question: not necessarily, no!  I don't think this feature was intended to create a new pattern for future features so much as it was intended to meet the needs around fiscal sponsorship as understood today.

Reply:
Ah, I see what you are suggesting @slifty. Yes, right now we have a case of "one special relationship", but if we shift to a case of "many special relationships" that is an opportune moment to make a more generic endpoint, such as `/changemakers/:changemakerId/:otherEntityName/:otherEntityId/:specialRelationshipEnumValue` or what have you.

Reply:
@bickelj @slifty If we did that future genericization, we would have a compatibility path by which we could transition the old `fiscalSponsors` way to the new `:specialRelationshipEnumValue` way?  I mean, I think the answer to that is "obviously yes", but I wanted to raise it here explicitly.  I understand that "premature generalization is the root of all evil", but I also want to make sure we're not locking the door against future evil... or something like that...

Reply:
Well the main issue is... this PR is a documentation PR for a feature that was already built / approved / merged / deployed.

Could we open up a new issue to capture the rest of this discussion, or Karl should we roll back that feature and re-implement?